pkg/config/BUILD

```diff
@@ -0,0 +1,34 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "cfg.pb.go",
+        "runtime.go",
+        "validater.go",
+    ],
+    deps = [
+        "//pkg/aspect:go_default_library",
+        "//pkg/attribute:go_default_library",
+        "//pkg/expr:go_default_library",
+        "@com_github_golang_protobuf//proto:go_default_library",
+        "@com_github_mitchellh_mapstructure//:go_default_library",
+        "@in_gopkg_yaml_v2//:go_default_library",
+    ],
+)
+
+go_test(
+    name = "config_test",
```

Comment:
nit/suggestion: call this "small_tests"... so the full path becomes `//pkg/config:small_tests`.